### ISSUE TITLE
Option to disable banner

### ISSUE THREAD

**Issue by OliverJAsh** [webdriverio/webdriverio#1982]
The [banner](https://github.com/webdriverio/webdriverio/blob/e84e743e858c764a4df8040fcff06fffbafbb515/lib/utils/Logger.js#L44) is a bit annoying as it shows once for each browser.

```
=======================================================================================
Selenium 2.0 / webdriver protocol bindings implementation with helper commands in nodejs.
For a complete list of commands, visit http://webdriver.io/api.html.
=======================================================================================
```

Do you think it's worth adding an option to hide it?

Thanks!

**Comment by christian-bromann** [comment]
I guess rather than adding another option, let's make sure it displays only once when running testrunner

**Comment by OliverJAsh** [comment]
Sounds sensible! Some people may not want it to show it all—would be good to have the option.

**Comment by christian-bromann** [comment]
merged into #878
